### ISSUE TITLE
Set the `/build/number` back to `0`

### ISSUE THREAD

**Issue by schuylermartin45** [conda-incubator/conda-recipe-manager#167]

**Comment by schuylermartin45** [comment]
No, this is part of the primary update process, not a flag. I guess you could think of it as "by default" reset the build number to zero.

**Comment by ForgottenProgramme** [comment]
So will it go like:
`crm bump-recipe update-version`?

**Comment by schuylermartin45** [comment]
No just `crm bump-recipe meta.yaml`.  The default update path does not require an additional command or flag or option.

**Comment by schuylermartin45** [comment]
Remember that this ticket is just part of what we plan to do. In the future `crm bump-recipe <file>` will update the version number, the SHA-256 (if applicable), dependencies, etc.
